pkg/cmd/release-payload-controller/release_creation_job_controller.go

```diff
@@ -171,9 +186,84 @@ func (c *ReleaseCreationJobController) sync(ctx context.Context, key string) err
 		return err
 	}
 
+	// Get/generate the release creation job result from the payload
+	releaseCreationJobResult := getReleaseCreationJobResult(originalReleasePayload, c.jobsNamespace)
+
+	// lookup the job
+	originalReleaseCreationJob, err := c.jobsLister.Jobs(releaseCreationJobResult.Coordinates.Namespace).Get(releaseCreationJobResult.Coordinates.Name)
+	if errors.IsNotFound(err) {
+		klog.V(4).Infof("Unable to locate release creation job: %s/%s", releaseCreationJobResult.Coordinates.Namespace, releaseCreationJobResult.Coordinates.Name)
+		err = nil
+		originalReleaseCreationJob = nil
+	}
+	if err != nil {
+		return err
+	}
+
+	if originalReleaseCreationJob != nil {
+		releaseCreationJobResult.Status = computeReleaseCreationJobStatus(originalReleaseCreationJob)
+		releaseCreationJobResult.Message = computeReleaseCreationJobMessage(originalReleaseCreationJob)
+	}
+
 	releasePayload := originalReleasePayload.DeepCopy()
 
-	klog.V(4).Infof("Syncing ReleasePayload: %s/%s", releasePayload.Namespace, releasePayload.Name)
+	klog.V(4).Infof("Syncing ReleaseCreationJobResult for ReleasePayload: %s/%s", releasePayload.Namespace, releasePayload.Name)
+	releasePayload.Status.ReleaseCreationJobResult = *releaseCreationJobResult
```

Comment:
!?  you both read and write this??

Reply:
Updated the 2 controllers to behave as follows:
The `ReleaseCreationJobController` write the following fields:
`.Status.ReleaseCreationJobResult.ReleaseCreationJobCoordinates.Namespace`
`.Status.ReleaseCreationJobResult.ReleaseCreationJobCoordinates.Name`

The `ReleaseCreationStatusController` write the following fields:
`.Status.ReleaseCreationJobResult.Status`
`.Status.ReleaseCreationJobResult.Message`